### PR TITLE
fix: add env var that allows to inject custom scripts into the head

### PR DESCRIPTION
**Pull request by Pagebakers**
…e head

## Motivation and Context

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: helm/cosmo/charts/studio/values.yaml [hunk 1]
```diff
@@ -118,6 +118,8 @@ probes:
 configuration:
   # -- The client ID of the slack app to use for slack app integration
   slackClientId: ""
+  # --- Custom scripts injected into the head of the studio, eg for analytics
```
Comment (StarpTech) [inline helm/cosmo/charts/studio/values.yaml:121]: You forgot to run `make docs`. Usually, the CI should catch it.

File: helm/cosmo/charts/studio/templates/deployment.yaml [hunk 2]
```diff
@@ -60,6 +60,11 @@ spec:
                 configMapKeyRef:
                   name: {{ include "studio.fullname" . }}
                   key: slackClientId
+            - name: CUSTOM_HEAD_SCRIPTS
+              valueFrom
+                configMapKeyRef:
+                  name: {{ include "studio.fullname" . }}
+                  key: customHeadScripts
```
Comment (StarpTech) [inline helm/cosmo/charts/studio/templates/deployment.yaml:67]: This reference to the config map. You need to a mapping here https://github.com/wundergraph/cosmo/blob/fix/custom-scripts/helm/cosmo/charts/studio/templates/config-map.yaml

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: helm/cosmo/charts/studio/values.yaml (hunk 1 above)
Comment (Pagebakers) [reply helm/cosmo/charts/studio/values.yaml:121]: I don't understand?

```make: Nothing to be done for `docs'.```

Comment (StarpTech) [reply helm/cosmo/charts/studio/values.yaml:121]: Ahh please use two dashes `--` :smile:

File: helm/cosmo/charts/studio/templates/deployment.yaml [hunk 3]
```diff
@@ -60,6 +60,11 @@ spec:
                 configMapKeyRef:
                   name: {{ include "studio.fullname" . }}
                   key: slackClientId
+            - name: CUSTOM_HEAD_SCRIPTS
+              valueFrom
```
Comment (thisisnithin) [inline helm/cosmo/charts/studio/templates/deployment.yaml:64]: Missing `:`

**Review by thisisnithin — CHANGES_REQUESTED** [review]

File: helm/cosmo/charts/studio/templates/deployment.yaml (hunk 3 above)
Comment (Pagebakers) [reply helm/cosmo/charts/studio/templates/deployment.yaml:64]: Good eye :D

File: helm/cosmo/charts/studio/values.yaml (hunk 1 above)
Comment (Pagebakers) [reply helm/cosmo/charts/studio/values.yaml]: Gotcha

**Review by thisisnithin — APPROVED** [review]

**Review by StarpTech — APPROVED** [review]
LGTM

**Pull request merged**